### PR TITLE
GH-17 - Add API Key sidebar tab + API key box and label

### PR DESCRIPTION
**Pull request by CozmoKitten**
* Issue: gh-17

### CODE REVIEW

File: gui/app.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from gui.frames.recruitment import RecruitmentFrame
 from gui.frames.construction import ConstructionFrame
 from gui.frames.settings import SettingsFrame
+from gui.frames.api_key_submit import APIKeyFrame
```
Comment (ItsNeil17) [inline gui/app.py:9]: rename file to api_key

File: gui/app.py [hunk 2]
```diff
@@ -23,8 +24,9 @@ def __init__(self):
         self.recruitment_frame = RecruitmentFrame(self)
         self.construction_frame = ConstructionFrame(self)
         self.settings_frame = SettingsFrame(self)
+        self.api_key_frame = APIKeyFrame(self)
 
-        self.current_frame = self.lookup_frame
+        self.current_frame = self.api_key_frame
```
Comment (ItsNeil17) [inline gui/app.py:29]: Add a check/function here that verifies if the API key is already saved & if it's valid. If any of those conditions aren't met, then show the api key frame.

**Review by ItsNeil17 — CHANGES_REQUESTED** [review]
Submit button is missing. Please correct the issues below:

**Comment by CozmoKitten** [comment]
I have made the requested changes; please review again.

Note I did not complete this:
> Add a check/function here that verifies if the API key is already saved & if it's valid. If any of those conditions aren't met, then show the api key frame.
as per my private conversation with @ItsNeil17

**Review by ItsNeil17 — APPROVED** [review]
LGTM

**Pull request merged**